src/AutoRest.CSharp.V3/Output/Models/Serialization/Json/JsonDictionarySerialization.cs

```diff
@@ -7,13 +7,13 @@ namespace AutoRest.CSharp.V3.Output.Models.Serialization.Json
 {
     internal class JsonDictionarySerialization : JsonSerialization
     {
-        public JsonDictionarySerialization(CSharpType? type, JsonSerialization valueSerialization)
+        public JsonDictionarySerialization(CSharpType type, JsonSerialization valueSerialization)
         {
             Type = type;
             ValueSerialization = valueSerialization;
         }
 
-        public CSharpType? Type { get; }
+        public CSharpType Type { get; }
```

Comment:
This was a fix because it was no longer being set to null in any situation. I'm assuming this was from a past code cleanup.